tools/market_health_reporter/market_health_reporter.py

```
@@ -189,4 +189,4 @@ def main():
             post_comment_to_issue(args.github_token, int(args.issue), REPO_NAME, output)
 
     except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)